website/docs/docs/contributing/building-a-new-adapter.md

```diff
@@ -316,6 +316,15 @@ While much of dbt's adapter-specific functionality can be modified in adapter ma
 
 </File>
 
+### Other files
+
+In order to enable the [dbt init command](/reference/commands/init), make sure to include a sample profile file. The filepath will be `dbt/adapters/<adapter_name>/sample_profiles.yml`. This will assure that users can create a new dbt project with the `dbt init` command. 
```

Comment:
```suggestion
In order to enable the [dbt init command](/reference/commands/init), make sure to include a sample profile file. The filepath will be `dbt/include/<adapter_name>/sample_profiles.yml`. This will assure that users can create a new dbt project with the `dbt init` command.
```